package.json

```diff
@@ -13,9 +13,15 @@
   "author": "Radar Labs, Inc.",
   "homepage": "https://radar.com",
   "license": "Apache-2.0",
+  "resolutions": {
+    "source-map": "^0.8.0-beta.0"
+  },
   "dependencies": {
     "@capacitor/core": "^3.0.1",
-    "radar-sdk-js": "^3.1.1"
+    "build": "^0.1.4",
```

Comment:
Will this end up adding these new deps to people's apps?  I guess a question I have is - why the new deps are being added?  Since it doesn't seem like these are used in the code changes. I suppose they're coming in from building the example app?

Reply:
I added these to work around a `You must provide the URL of lib/mappings.wasm by calling SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using SourceMapConsumer` error that popped up in some recent Node change, but I'll check whether I removing the `build` and `run` ones will continue to work.

Reply:
Yeah, it looks like `npm` keeps re-insertings the `build` and `run` dependencies, even after I delete them from `package.json`.

Reply:
What `npm` command is adding them to package.json? It usually doesn't touch `package.json` unless you're doing `npm install` or `npm update`

Is it just the `run_example.sh` code where it explicitly calls:
```
npm install run build
npm install -g @ionic/cli
```

Or something else?

The reason I'm squinting hard at adding these is because dependency conflicts are one of the biggest sources of pain for users of the sdk.

Reply:
Yes, it's `npm install`. My `run_example` script more or less does what the CircleCI workflow does, but CircleCI doesn't save any changes to `package.json`, so I'm not sure what it's doing.

Reply:
I just removed them, and even though `npm` keeps re-adding them when I `npm install` locally, CircleCI seems to have no problem without them.